app/models/code-list.js

```diff
@@ -4,4 +4,11 @@ export default class CodeListModel extends Model {
   @attr label;
   @hasMany('code-list-option', { inverse: 'codeList' }) codeListOptions;
   @hasMany('mapping', { inverse: 'codeList' }) mappings;
+
+  get readableLabel() {
```

Comment:
Not sure if this was a hard requirement or not but I think it's a bit confusing that the label that is visible everywhere doesn't match how the user enters it in the form. Switching from the details to the edit page now "changes" the casing. I think I would prefer just seeing the 'real' casing everywhere. This is just my opinion of course, so feel free to ignore this.

Reply:
Not a requirement, just thought it was more readable that way but I didn't think about the confusing part indeed ^^' will get rid of it :ok_hand: